network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -52,30 +52,37 @@ <h2 class="h1-heading col-md-10">{{product.name}}</h2>
           <div class="d-block primary-info">
             <div class="d-flex justify-content-between">
               <div class="d-flex flex-column flex-md-row justify-content-md-between w-100">
-                <h3 class="h3-heading">Minimum Security Standards</h3>
+                <h3 class="h3-heading">{% trans "Minimum Security Standards" %}</h3>
               </div>
             </div>
             <div class="primary-info-description mb-3">
               <p class="body">
-                Five basic steps every company should take to protect consumer privacy.
-                <a href="/privacynotincluded/about/meets-minimum-security-standards">Learn more.</a>
+                {% trans "Five basic steps every company should take to protect consumer privacy." %}
+                <a href="/privacynotincluded/about/meets-minimum-security-standards">{% trans "Learn more." %}</a>
               </p>
             </div>
           </div>
 
           <div class="details mt-2">
             <div class="criterion rating">
-              {% include "fragments/product_criterion_primary_info.html"  with  label="Overall Security Rating" %}
+              {% trans "Overall Security Rating" as overall_security_rating %}
+              {% include "fragments/product_criterion_primary_info.html"  with label=overall_security_rating %}
               <div>
-                <span class="security-score mb-0">{{security_score}}</span><span class="full-security-score mb-0">/{{full_security_score}}</span>
-                <img src="/_images/buyers-guide/score/icon-star.svg" width="20" height="20" alt="star">
+                <span class="security-score mb-0">{% localize on %}{{security_score}}{% endlocalize %}</span><span class="full-security-score mb-0">/{{full_security_score}}</span>
```

Comment:
This is a tag I've not seen before, what does it do? (this might be a fun one, because this isn't technically "a number" and more something that represents a value on a scale akin to "X out of Y")

Reply:
This renders the number according to the locale formatting rules. For what matters here, it will render `2.5` in English, but `2,5` in French

Reply:
Since it’s a single value here, we can be smarter and use the template filter instead of template tags (updated the code) https://docs.djangoproject.com/en/3.0/topics/i18n/formatting/#std:templatefilter-localize

`{{security_score|localize}}`